vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -58,6 +57,19 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - cp /etc/kubernetes/admin.conf /home/{{ IMAGE_USERNAME }}/.kube/config
       - chown {{ IMAGE_USERNAME }}:{{ IMAGE_USERNAME }} /home/{{ IMAGE_USERNAME }}/.kube/config
     files:
+      - path: /usr/local/bin/retrieve.kubeadm.kubelet.configs.sh
+        owner: root:root
+        permissions: '0755'
+        content: |
+            #!/bin/bash
+            curl -sSL -w \"%{http_code}\" \"https://raw.githubusercontent.com/kubernetes/release/v0.2.7/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service\" | \
+              sed \"s:/usr/bin:/usr/local/bin:g\" > /tmp/kubelet.service;http_status=\$(cat /tmp/kubelet.service | tail -n 1)
+            if [ \"\${http_status}\" != \"200\" ]; then echo \"Unable to retrieve kubelet.service unit file\";exit 1;else cat /tmp/kubelet.service|\
+              sed '\$d' > /etc/systemd/system/kubelet.service;fi
+            curl -sSL -w \"%{http_code}\" \"https://raw.githubusercontent.com/kubernetes/release/v0.2.7/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf\" | \
+              sed \"s:/usr/bin:/usr/local/bin:g\"> /tmp/10-kubeadm.conf;http_status=\$(cat /tmp/10-kubeadm.conf | tail -n 1)
+            if [ \"\${http_status}\" != \"200\" ]; then echo \"Unable to retrieve 10-kubeadm.conf file\";exit 1;else cat /tmp/10-kubeadm.conf | \
+              sed '\$d' > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf;fi
```

Comment:
could you maybe make this file take two inputs, the URL and the destination file, and then call it twice in the commands, once for kubelet and once for kubeadm config files ? You could also break the unreadable lines into nicer looking if structures.

Reply:
@Xenwar will you refactor this ?

Reply:
Can parameterize it. 
But, the lines are no longer breakable without making less readable. Some of the urls are quite long

Reply:
if you parameterize it it removes the long urls from the script

Reply:
it would also be more readable if you did not have the if else fi in the same line, but break it down, with indentation etc.

Reply:
Done